src/pages/audio-rewards-page/store/sagas.ts

```diff
@@ -244,7 +244,8 @@ export function* watchFetchUserChallenges() {
     const currentUserId: number = yield select(getUserId)
 
     try {
-      const userChallenges: UserChallenge[] = yield apiClient.getUserChallenges(
+      const userChallenges: UserChallenge[] = yield call(
```

Comment:
can we try using yield* and typed-redux-saga for this file? does that mess up testing?

Reply:
Yeah we could - this branch predated that change but this file is probably a great candidate to start converting! I'll try it out and see how it affects testing